lib/ansible/modules/cloud/vmware/vmware_dvs_portgroup.py

```diff
@@ -23,94 +24,143 @@
+        required: False
+    private_vlan_id:
+        description:
+            - The secondary private VLAN ID.
+            - The secondary private VLAN ID need to be configured on the dvSwitch first.'
+        type: int
+        version_added: '2.8'
+    vlan_trunk_range:
+        description:
+            - The VLAN trunk range that should be configured with the portgroup.
+            - 'This can be a combination of multiple ranges and numbers, example: [ 1-200, 205, 400-4094 ].'
+            - The default VLAN trunk range in the vSphere Client is [ 0-4094 ].
+        type: list
+        version_added: 2.8
     vlan_trunk:
         description:
             - Indicates whether this is a VLAN trunk or not.
+            - Deprecated. Will be removed in 2.11.
         required: False
         default: False
         type: bool
         version_added: '2.5'
-    network_policy:
+    security:
         description:
             - Dictionary which configures the different security values for portgroup.
             - 'Valid attributes are:'
-            - '- C(promiscuous) (bool): indicates whether promiscuous mode is allowed. (default: false)'
+            - '- C(promiscuous_mode) (bool): indicates whether promiscuous mode is allowed. (default: false)'
             - '- C(forged_transmits) (bool): indicates whether forged transmits are allowed. (default: false)'
             - '- C(mac_changes) (bool): indicates whether mac changes are allowed. (default: false)'
         required: False
         version_added: '2.5'
         default: {
-            promiscuous: False,
+            promiscuous_mode: False,
             forged_transmits: False,
             mac_changes: False,
         }
-    teaming_policy:
+        aliases: ['network_policy', 'security_policy']
+    teaming:
         description:
             - Dictionary which configures the different teaming values for portgroup.
             - 'Valid attributes are:'
-            - '- C(load_balance_policy) (string): Network adapter teaming policy. (default: loadbalance_srcid)'
+            - '- C(load_balancing) (string): Network adapter teaming policy. (default: loadbalance_srcid)'
             - '   - choices: [ loadbalance_ip, loadbalance_srcmac, loadbalance_srcid, loadbalance_loadbased, failover_explicit]'
             - '   - "loadbalance_loadbased" is available from version 2.6 and onwards'
-            - '- C(inbound_policy) (bool): Indicate whether or not the teaming policy is applied to inbound frames as well. (default: False)'
+            - '- C(network_failure_detection) (string): Network failure detection. (default: link_status_only)'
+            - '   - choices: [ link_status_only, beacon_probing ]'
             - '- C(notify_switches) (bool): Indicate whether or not to notify the physical switch if a link fails. (default: True)'
-            - '- C(rolling_order) (bool): Indicate whether or not to use a rolling policy when restoring links. (default: False)'
```

Comment:
`rolling_order` should not be removed, it is still a valid setting in vSphere 6.7 API:
https://code.vmware.com/apis/358#/doc/vim.host.NetworkPolicy.NicTeamingPolicy.html#rollingOrder

Reply:
Ok, sounds good, i get that :).

Reply:
I added an alias